### DISCUSSION TITLE
Incorporate Utility Classes from Spring Boot Cli

### DISCUSSION THREAD

**Discussion by pdavie** [spring-projects/spring-shell#822] [Ideas]
I am investiagting implementing Spring Shell for a project and there are some useful utility classes available in Spring Boot CLI, e.g. OptionParsingCommand and OptionHandler which would be convenient to use.  Has there been any consideration in moving these classes across into Spring Shell to reduce the implementation overhead for other Spring Shell implementers?

**Comment by jvalkeal** [comment]
Could you tell a bit more what kind of functionality you are missing from `spring-shell` vs. `boot-cli` what comes from i.e. `OptionParsingCommand` as I don't quite understand what you are trying to accomplish.
